AstroAdapt.GraphQL/ComponentExtensions.cs

```diff
@@ -6,36 +6,14 @@ namespace AstroAdapt.GraphQL
     /// <summary>
     /// Extend component to enable fetching the image.
     /// </summary>
-    public class ComponentExtensions : ObjectTypeExtension<Component>
+    [ExtendObjectType(typeof(Component))]
```

Comment:
I simplified this type extension. You can take any class and annotate it with the `ExtendObjectTypeAttribute` and it will merge public methods and props into the target type.